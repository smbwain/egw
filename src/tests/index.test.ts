import './001-config-core';
import './002-subcontest';
