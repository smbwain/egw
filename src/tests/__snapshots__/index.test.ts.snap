// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`config core should fill config 1`] = `
Object {
  "a": "555",
  "b": "222",
  "c": Object {
    "d": "666",
    "e": "444",
  },
}
`;

exports[`config core should fill config env 1`] = `
Object {
  "a": "555",
  "b": "222",
  "c": Object {
    "d": "666",
    "e": "444",
  },
}
`;

exports[`config core should merge config 1`] = `
Object {
  "a": "666",
  "b": "222",
  "c": Object {
    "d": "333",
    "e": "777",
    "i": "888",
  },
  "f": Object {
    "g": "555",
  },
  "j": Object {
    "k": "999",
  },
  "l": "000",
}
`;
